src/components/ZclDomainClusterView.vue

```diff
@@ -539,6 +568,16 @@ export default {
         { label: 'Server', client: false, server: true },
         { label: 'Client & Server', client: true, server: true },
       ],
+      optionsForCheckboxes: [
+        {
+          label: 'Client',
```

Comment:
Avoid using hardcoded strings in the code. Define it at one place and re-use like the db-enum file we have.